### ISSUE TITLE
[0.10.0-beta2] /root/.config/lxc/config.yml missing

### ISSUE THREAD

**Issue by shantanugadgil** [sl1pm4t/terraform-provider-lxd#79]
**terraform version**
Terraform v0.9.11

**terraform-provider-lxd version**
0.10.0-beta2
(https://github.com/sl1pm4t/terraform-provider-lxd/releases/download/v0.10.0-beta2/terraform-provider-lxd_v0.10.0-beta2_linux_amd64.tar.gz)

**OS Details**
CentOS Linux release 7.3.1611  (with latest updates)

On my existing setup, I replaced the old provider plugin and ran `# terraform plan`

I got the following error:
```
* provider.lxd: Could not read the lxc config: [/root/.config/lxc/config.yml]. Error: Unable to read the configuration file: open /root/.config/lxc/config.yml: no such file or directory
```

I touched that file and the error went away and plan and apply succeeded.

Regards,
Shantanu

**Comment by jtopjian** [comment]
I can confirm this.

It looks like [this](https://github.com/lxc/lxd/blob/342a991e0e6e789d22b1f3b692e70b7ad818605c/config.go#L88) is the `LoadConfig` to use, but [this](https://github.com/lxc/lxd/blob/83ee3e9a46de9bc3ef9804a4a04c7ae3d7211fda/lxc/config/file.go#L16) one is being used instead.

There's a chain of changes that are triggered once making that change. Not sure the best way to handle those.

**Comment by jtopjian** [comment]
This should do it:

```go
configDir := d.Get("config_dir").(string)
configPath := os.ExpandEnv(path.Join(configDir, "config.yml"))
if conf, err := lxd_config.LoadConfig(configPath); err != nil {
  config = &lxd_config.DefaultConfig
} else {
  config = conf
}
```

**Comment by sl1pm4t** [comment]
Fixed by #80
